soh/src/overlays/actors/ovl_En_Dnt_Jiji/z_en_dnt_jiji.c

```diff
@@ -101,6 +101,9 @@ void EnDntJiji_Destroy(Actor* thisx, PlayState* play) {
 }
 
 void EnDntJiji_SetFlower(EnDntJiji* this, PlayState* play) {
+    // SOH: Due to removed object dependencies, parent was still NULL when Init was called. In order to proper set
```

Comment:
dummy suggestion to trigger build
```suggestion
    // SOH: Due to removed object dependencies, parent was still NULL when Init was called. In order to properly set
```